connection.go

```diff
@@ -471,26 +445,26 @@ type connectionClient struct {
 	conn *Connection
 }
 
-var _ rpc.GenericClient = connectionClient{}
+var _ GenericClient = connectionClient{}
 
 func (c connectionClient) Call(ctx context.Context, s string, args interface{}, res interface{}) error {
-	return c.conn.DoCommand(ctx, s, func(rawClient rpc.GenericClient) error {
-		tags, ok := logger.LogTagsFromContext(ctx)
+	return c.conn.DoCommand(ctx, s, func(rawClient GenericClient) error {
+		tags, ok := c.conn.tagsFunc(ctx)
 		if ok {
-			rpcTags := make(rpc.CtxRpcTags)
+			rpcTags := make(CtxRpcTags)
 			for key, tagName := range tags {
 				if v := ctx.Value(key); v != nil {
```

Comment:
This code isn't new to this port, and I didn't write it, but yeah we are taking the debug logger tags from the context and adding them back in as rpc tags.